### ISSUE TITLE
Windows-error with HOME - variable

### ISSUE THREAD

**Issue by sulheim** [a-slide/pycoQC#57]
**Describe the bug**
When running pycoQC v2.2.2 on Windows I get the following error message:
```
PARSE CONFIGURATION FILE
Traceback (most recent call last):
  File "c:\users\snorres\appdata\local\continuum\anaconda3\envs\pycoqc\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\snorres\appdata\local\continuum\anaconda3\envs\pycoqc\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\snorres\AppData\Local\Continuum\Anaconda3\envs\pycoQC\Scripts\pycoQC.exe\__main__.py", line 9, in <module>
  File "c:\users\snorres\appdata\local\continuum\anaconda3\envs\pycoqc\lib\site-packages\pycoQC\cli.py", line 169, in main_pycoQC
    title=args.title)
  File "c:\users\snorres\appdata\local\continuum\anaconda3\envs\pycoqc\lib\site-packages\pycoQC\cli.py", line 186, in generate_report
    config_dict = parse_config_file(config)
  File "c:\users\snorres\appdata\local\continuum\anaconda3\envs\pycoqc\lib\site-packages\pycoQC\cli.py", line 280, in parse_config_file
    home=os.environ['HOME']
  File "c:\users\snorres\appdata\local\continuum\anaconda3\envs\pycoqc\lib\os.py", line 725, in __getitem__
    raise KeyError(key) from None
KeyError: 'HOME'
```

**To Reproduce**
On Windows:
```shell
pycoQC --file sequencing_summary.text
```

**Expected behavior**
Run as normal

**Desktop (please complete the following information):**
 - Windows 10
 - Chrome
 - 2.2.2

**Additional context**
The issue is solved by adding "HOME" as an environmental variable with 
```shell
set HOME=%USERPROFILE%
```
as described in https://stackoverflow.com/questions/14742064/python-os-environhome-works-on-idle-but-not-in-a-script

**Comment by a-slide** [comment]
Hi @sulheim,
Thanks for raising the issue and for the detailed error reporting.
I think I fixed the issue in pycoQC directly and just published it under version 2.2.3.
If you update with pip you should be able to get the last version
Please, let me know if it works now without having to set the home environment manually ?
Thanks

**Comment by sulheim** [comment]
It works, thanks!
